daemon/cmd/daemon_main.go

```diff
@@ -580,6 +580,9 @@ func init() {
 	flags.String(option.MaglevHashSeed, maglev.DefaultHashSeed, "Maglev cluster-wide hash seed (base64 encoded)")
 	option.BindEnv(option.MaglevHashSeed)
 
+	flags.Uint(option.MaglevMaxBackends, maglev.DefaultMaxBackends, "Maglev maximum number of backends per service")
```

Comment:
Maybe we could improve the wording here, as users might think that this setting determines max backends count which is no longer true (https://github.com/cilium/cilium/pull/14622/files#diff-b0db115015c51eec25d33620ff0a26546987ad1282597ef71fac29e50c9e3c28R92).

Reply:
Ah good catch. I was planning to update that and did it in other places, but forgot this one.